### PR TITLE
Load moar csvs

### PR DESCRIPTION
**Pull request by 7maples**
Pivotal Cards: [89140146](https://www.pivotaltracker.com/story/show/89140146), [89139764](https://www.pivotaltracker.com/story/show/89139764), [89139702](https://www.pivotaltracker.com/story/show/89139702), [89139630](https://www.pivotaltracker.com/story/show/89139630), [89139048](https://www.pivotaltracker.com/story/show/89139048)

All of these cards are geared towards adding CSV loading capabilities for the following files:
* locality_early_vote_site.txt
* custom_ballot_ballot_response.txt
* custom_ballot.txt
* contest_result.txt
* ballot_line_result.txt

The data processor can now upload this data into individual tables and make changes (i.e. boolean values) where appropriate.

### CODE REVIEW

File: src/vip/data_processor/validation/csv.clj [hunk 1]
```diff
@@ -145,3 +145,8 @@
 (def load-precinct-split-polling-locations (csv-loader "precinct_split_polling_location.txt" :precinct-split-polling-locations))
 (def load-precinct-electoral-districts (csv-loader "precinct_electoral_district.txt" :precinct-electoral-districts))
 (def load-precinct-early-vote-sites (csv-loader "precinct_early_vote_site.txt" :precinct-early-vote-sites))
+(def load-locality-early-vote-sites (csv-loader "locality_early_vote_site.txt" :locality-early-vote-sites))
+(def load-custom-ballot-ballot-responses (csv-loader "custom_ballot_ballot_response.txt" :custom-ballot-ballot-responses))
+(def load-custom-ballots (csv-loader "custom_ballot.txt" :custom-ballots))
+(def load-contest-results (csv-loader "contest_result.txt" :contest-results))
+(def load-ballot-line-results (csv-loader "ballot_line_result.txt" :ballot-line-results))
```
Comment (tie-rack) [inline src/vip/data_processor/validation/csv.clj]: `load-contest-results` and `load-ballot-line-results` need `booleanize` for their boolean columns.

Comment (7maples) [inline src/vip/data_processor/validation/csv.clj]: Doh! added

File: src/vip/data_processor/validation/csv.clj [hunk 2]
```diff
@@ -145,3 +145,10 @@
 (def load-precinct-split-polling-locations (csv-loader "precinct_split_polling_location.txt" :precinct-split-polling-locations))
 (def load-precinct-electoral-districts (csv-loader "precinct_electoral_district.txt" :precinct-electoral-districts))
 (def load-precinct-early-vote-sites (csv-loader "precinct_early_vote_site.txt" :precinct-early-vote-sites))
+(def load-locality-early-vote-sites (csv-loader "locality_early_vote_site.txt" :locality-early-vote-sites))
+(def load-custom-ballot-ballot-responses (csv-loader "custom_ballot_ballot_response.txt" :custom-ballot-ballot-responses))
+(def load-custom-ballots (csv-loader "custom_ballot.txt" :custom-ballots))
+(def load-contest-results (csv-loader "contest_result.txt" :contest-results
+                                      (booleanize "entire_district")))
+(def load-ballot-line-results (csv-loader "ballot_line_result.txt" :ballot-line-results
+                                          (booleanize "entire_district")))
```
Comment (tie-rack) [inline src/vip/data_processor/validation/csv.clj]: One more... for "victorious"

File: src/vip/data_processor/validation/csv.clj [hunk 3]
```diff
@@ -145,3 +145,11 @@
...
 (def load-precinct-electoral-districts (csv-loader "precinct_electoral_district.txt" :precinct-electoral-districts))
 (def load-precinct-early-vote-sites (csv-loader "precinct_early_vote_site.txt" :precinct-early-vote-sites))
+(def load-locality-early-vote-sites (csv-loader "locality_early_vote_site.txt" :locality-early-vote-sites))
+(def load-custom-ballot-ballot-responses (csv-loader "custom_ballot_ballot_response.txt" :custom-ballot-ballot-responses))
+(def load-custom-ballots (csv-loader "custom_ballot.txt" :custom-ballots))
+(def load-contest-results (csv-loader "contest_result.txt" :contest-results
+                                      (booleanize "entire_district")))
+(def load-ballot-line-results (csv-loader "ballot_line_result.txt" :ballot-line-results
+                                          (booleanize "entire_district")
+                                          (booleanize "victorious"))
```
Comment (tie-rack) [inline src/vip/data_processor/validation/csv.clj]: Now you're missing a closing parenthesis here.

**Pull request merged**